spec/models/instance_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe Instance, type: :model do
+RSpec.describe Instance do
```

Comment:
It's an RSpec feature, for anything inside the spec folder... see the rspec-rails documentation.